pecos/xmc/xlinear/model.py

```diff
@@ -392,10 +386,17 @@ def reconstruct_model(cls, parent_model, child_models, Y_ids_of_child_models=Non
                 new_W = new_W[:, inv_permuted_Y_label]
                 new_C.indices = permuted_Y_label[new_C.indices]
             model_chain.append(
-                MLModel(C=new_C, W=new_W, bias=child_models[0].model.model_chain[d].bias)
+                MLModel(
+                    C=new_C,
+                    W=new_W,
+                    bias=child_models[0].model.model_chain[d].bias,
+                    pred_params=child_models[0].model.get_pred_params().model_chain[d],
```

Comment:
We should be explicit in the docstring that the final model is taking `pred_params` from the first child model.